CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.1)
-PROJECT(dot11decrypt)
 
-FIND_PACKAGE(libtins REQUIRED)
+SET(CMAKE_CXX_STANDARD 11)
+SET(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+PROJECT(dot11decrypt)
```

Comment:
Why did this move down here?

Reply:
The example I saw that worked for me had it like this, I can move it back up